local.Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Builds images used by the bridge using locally built binaries.
+#
+# In particular, it can be used to build Substrate nodes and bridge relayers. The binary that gets
+# built can be specified with the `PROJECT` build-arg. For example, to build the `substrate-relay`
+# you would do the following:
+#
+# `docker build ./target -f local.Dockerfile -t local/substrate-relay --build-arg=PROJECT=substrate-relay`
+#
+# See the `deployments/README.md` for all the available `PROJECT` values.
+
+FROM docker.io/library/ubuntu:20.04 as runtime
+
+USER root
+WORKDIR /home/root
+
+# show backtraces
+ENV RUST_BACKTRACE 1
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN set -eux; \
+	apt-get update && \
+	apt-get install -y --no-install-recommends \
+        curl ca-certificates libssl-dev && \
+    update-ca-certificates && \
+	groupadd -g 1000 user && \
```

Comment:
I'm using ubuntu 23.04 and gid 1000 doesn't work for me. Could we use another value here ? 1001 or 2000 for example ?

Reply:
Changed to 1001